.circleci/config.yml

```diff
@@ -231,10 +231,10 @@ workflows:
           requires:
             - build_icon-library
           filters:
-            branches:
-              ignore:
-                - master
-                - latest
+           branches:
```

Comment:
Should there be two spaces rather than one?

Reply:
Good spot, will fix